couchbase/json-buckets.yml

```diff
@@ -0,0 +1,968 @@
+        description: Number of other producers
+
+      - name: ep.dcp.other.total.bytes
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_other_total_bytes[0]
+        type: long_gauge
+        label: other DCP
+        unit: bytes/second
+        description: Number bytes per second being sent for other DCP connections
+
+      - name: ep.dcp.xdcr.backoff
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_xdcr_backoff[0]
+        type: long_gauge
+        label: backoffs for xdcr DCP
+        description: Number of backoffs for xdcr DCP connections
+
+      - name: ep.dcp.xdcr.count
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_xdcr_count[0]
+        type: long_gauge
+        label: xdcr DCP
+        description: Number of xdcr DCP connections
+
+      - name: ep.dcp.xdcr.items.remaining
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_xdcr_items_remaining[0]
+        type: long_gauge
+        label: remaining xdcr items
+        description: Number of xdcr items remaining to be sent
+
+      - name: ep.dcp.xdcr.items.sent
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_xdcr_items_sent[0]
+        type: long_gauge
+        label: sent xdcr items
+        description: Number of xdcr items sent
+
+      - name: ep.dcp.xdcr.producer.count
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_xdcr_producer_count[0]
+        type: long_gauge
+        label: xdcr producers
+        description: Number of xdcr producers
+
+      - name: ep.dcp.xdcr.total.bytes
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_dcp_xdcr_total_bytes[0]
+        type: long_gauge
+        label: xdcr DCP size
+        unit: bytes/second
+        description: Number bytes per second being sent for xdcr DCP connections
+
+      - name: ep.diskqueue.drain
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_diskqueue_drain[0]
+        type: long_gauge
+        label: drained disk queue items
+        description: Total Drained items on disk queue
+
+      - name: ep.diskqueue.fill
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_diskqueue_fill[0]
+        type: long_gauge
+        label: enqueued disk queue items
+        description: Total enqueued items on disk queue
+
+      - name: ep.diskqueue.items
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_diskqueue_items[0]
+        type: long_gauge
+        label: disk queue items
+        description: Total number of items waiting to be written to disk
+
+      - name: ep.flusher.todo
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ep_flusher_todo[0]
+        type: long_gauge
+        label: slusher todo
+        description: Number of items currently being written
```

Comment:
Doesn't match name and label.

Reply:
Yes, but that the label i get
https://raw.githubusercontent.com/couchbase/ep-engine/master/docs/stats.org

ep_vb_backfill_queue_size          | Number of items in vb backfill queues  |
| ep_flusher_todo                    | Number of items currently being        |
|                                    | written                                |
| ep_flusher_state                   | Current state of the flusher thread